docs/source/whatsnew/1.0.2.txt

```diff
@@ -36,6 +36,12 @@ Enhancements
   produced a True on N or more days in the previous ``window_length``
   days (:issue:`1367`).
 
+- Remove risk-free rate from sharpe ratio calculation. (:issue:`853`).
+
+- Use external library empyrical for risk calculations. Empyrical unifies risk
+  metric calculations between pyfolio and zipline. Empyrical adds custom
+  annualization options for returns of custom frequencies.
```

Comment:
Looks like a good place to reference https://github.com/quantopian/zipline/issues/855.